spec/requests/posts_spec.rb

```diff
@@ -2,6 +2,33 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    it 'returns https success' do
+      user = User.create(name: 'Austin', id: '1')
+      get user_post_path(user)
```

Comment:
- [ ] After adding the gem to Gemfile and running the bundler, this test still fails. You need to specify the correct route: `user_posts_path(user)`

  ![image](https://github.com/Obimbo07/tdn/assets/81468675/8aa0547e-55f4-4490-8591-ad4cc7bdd0a5)